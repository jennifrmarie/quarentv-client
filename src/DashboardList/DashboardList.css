.DashboardList_list {
    list-style: none;
    margin: 10px;
}

li {
    display: block;
}