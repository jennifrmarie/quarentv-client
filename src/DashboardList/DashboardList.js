import React, { Component } from 'react'
import AppContext from '../AppContext'
import './DashboardList.css'

import ListNav from '../ListNav/ListNav'

export default class DashboardList extends Component {
    static contextType = AppContext

    constructor(props) {
        super(props);
        this.state = {
          isHidden: true,
          showButton: true,
        };
      }

      toggleHidden () {
        this.setState({
          isHidden: !this.state.isHidden,
          showButton: this.state.showButton
        })
      }




    render() {
        return (
            <div>
              {/* <button onClick={this.toggleHidden.bind(this)} >
                Show what you've previously watched
              </button> */}
              <ListNav />
              {/* {!this.state.isHidden && <WatchList /> } */}


            </div>
        )
    }
}
