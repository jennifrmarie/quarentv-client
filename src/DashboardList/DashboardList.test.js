import React from 'react';
import ReactDOM from 'react-dom';
import DashboardList from './DashboardList';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<DashboardList />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<DashboardList name="Messages" unread={4}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });