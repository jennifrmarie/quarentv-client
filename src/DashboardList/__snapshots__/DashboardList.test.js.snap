// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <div
    className="DashboardListNav"
  >
    <ul
      className="DashboardList_list"
    >
      <h2
        className="list__title"
      >
        Get to work!
      </h2>
    </ul>
    <section
      className="scoreboard_section"
    >
      <h2
        className="scoreboard__title"
      >
        Scoreboard
      </h2>
      <ul
        className="scoreboard_list_section"
      >
        <li
          className="score__div"
        >
          Score: 
          0
        </li>
        

        

        <li
          className="badge_message"
        >
          

        </li>
        <li
          className="next_message"
        />
      </ul>
    </section>
  </div>
</div>
`;
